pkgs/top-level/all-packages.nix

```diff
@@ -18609,7 +18609,7 @@ in
 
   spotifyd = callPackage ../applications/audio/spotifyd {
     withALSA = stdenv.isLinux;
-    withPulseAudio = config.pulseaudio or true;
```

Comment:
This seems wrong to me. We should enable pulseaudio by default.

Reply:
sorry for the late response. I think we should not enable pulseaudio on darwin, so `config.pulseaudio or stdenv.isLinux` may be an alternative; but since spotifyd is broken on darwin, and I can't manage to fix it, it does not matter right now.